plugins/adr/src/components/EntityAdrContent/EntityAdrContent.tsx

```diff
@@ -82,7 +80,8 @@ export const EntityAdrContent = (props: {
   }, [entity, scmIntegrations]);
 
   const selectedAdr =
-    adrList.find(adr => adr.name === searchParams.get('record'))?.name ?? '';
+    adrList.find(adr => adr.name === searchParams.get('record'))?.path ?? '';
```

Comment:
I don't think this will work the way you want it since search params are currently being set to the selected adr name and you're also comparing `name` to `path` here

Reply:
Record attribute (line 83,87,147) use "name" of adr, but I think it's necessary to replace all occurrence by the "path" to prevent eventually duplicate name.

Reply:
I think you still need to replace `name` here with `path` for the duplicate name scenario: 
`adrList.find(adr => adr.path === searchParams.get('record'))?.path` and also update [here](https://github.com/backstage/backstage/pull/16430/files#diff-4d0e73660e5f8af857debc93fa580fb03939e07352d1523e36a06a9f22451ac9R125) to set the query param to the path too